src/pages/_document.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react'
 import Document, { Head, Main, NextScript } from 'next/document'
 import { ServerStyleSheet } from 'styled-components'
+import sprite from 'svg-sprite-loader/runtime/sprite.build'
+import '../static/icons.svg?sprite'
```

Comment:
Do we really need this import?

Reply:
Yes we do, the sprite is not working otherwise

Reply:
But then what does line 4 do?